src/init.cpp

```diff
@@ -1972,11 +1972,11 @@ bool AppInitMain(InitInterfaces& interfaces)
 
         uiInterface.InitMessage(_("Loading block index..."));
 
-        LOCK(cs_main);
-
         do {
             const int64_t load_block_index_start_time = GetTimeMillis();
+            bool is_coinsview_empty;
             try {
+                LOCK(cs_main);
```

Comment:
why don't we have these changes https://github.com/dashpay/dash/pull/4494

Reply:
that's because we have no `RewindBlockIndex` which is used to fix issues on post-segwit node updates